packages/enhanced-img/package.json

```
@@ -49,5 +49,9 @@
 		"typescript": "^5.3.3",
 		"vite": "^5.2.8",
 		"vitest": "^1.6.0"
+	},
+	"peerDependencies": {
+		"svelte": "^4.0.0 || ^5.0.0-next.0",
+		"vite": "^5.0.0"
```

let's avoid having this break when Svelte 6 comes out
```suggestion
		"vite": ">= 5.0.0"
```